surveys/2022-annual-survey/questions.md

```diff
@@ -375,7 +405,7 @@ Statements:
 > This question can help confirm or challenge our assumptions and see how they change over time.
 
 
-### Which of the following statements apply to your team's experience using Rust at work.
+### Which of the following statements apply to your experience using Rust at work.
```

Comment:
I'm unsure about removing reference to teams. I personally want the respondent to respond as a representative of their team rather than for their personal viewpoints.

Reply:
Yeah, I agree we want that. My thought was that this too subtle a nudge for it to make much difference and that it makes the questions a bit too confusing to parse